src/services/cohortCreation/index.ts

```diff
@@ -169,7 +169,7 @@ const createCohortGroup = async (jsonQuery: string, cohortName?: string): Promis
     name: cohortName,
     type: GroupTypeKind._person,
     actual: true,
-    quantity: patientIds.length,
+    quantity: uniq(patientIds).length,
```

Comment:
I didn't read the whole file yet but why are there duplicate ids in the first place?

Reply:
because it seems we have patients belonging to multiple organizations (a patient can have encounters in several organizations)

Reply:
sorry my diagnostic was wrong in the first place. It should be more obvious where the issue was now.